kickoff-web.py

```diff
@@ -33,6 +33,7 @@
     app.config['SQLALCHEMY_DATABASE_URI'] = options.db
     app.config['DEBUG'] = True
     app.config['SECRET_KEY'] = 'NOT A SECRET'
+    app.config['VERSION_FILE'] = '/app/version.json'
```

Comment:
Hmmm, the current deploy doesn't live in docker, and the path is different. Can you make it relative or calculated run-time?